### DISCUSSION TITLE
Error in container logs

### DISCUSSION THREAD

**Discussion by S54Mpower** [jertel/vuegraf#127] [General]
Hello Jertel,

have you came across logs as below? This are the logs I am getting from the docker container for vuegraf. Still did not figure out what the issue is, thanks for help!
```
Traceback (most recent call last):
  File "/opt/vuegraf/vuegraf.py", line 239, in <module>
    config = json.load(configFile)
             ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/__init__.py", line 293, in load
    return loads(fp.read(),
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/json/decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 2 column 15 (char 15)
```

**Comment by jertel** [comment]
It's telling you there's something wrong with your vuegraf.json config file. Check it for syntax issues.

**Reply by S54Mpower** [reply]
Thanks, there was a syntax error indeed. 

Now I progressed to:

```
Traceback (most recent call last):
  File "/opt/vuegraf/vuegraf.py", line 287, in <module>
    influx.create_database(config['influxDb']['database'])
  File "/usr/local/lib/python3.11/site-packages/influxdb/client.py", line 746, in create_database
    self.query("CREATE DATABASE {0}".format(quote_ident(dbname)),
  File "/usr/local/lib/python3.11/site-packages/influxdb/client.py", line 521, in query
    response = self.request(
               ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/influxdb/client.py", line 378, in request
    raise InfluxDBClientError(err_msg, response.status_code)
influxdb.exceptions.InfluxDBClientError: 401: {"code":"unauthorized","message":"Unauthorized"}
```

**Reply by jertel** [reply]
Please wrap your code/error pastes in triple backticks. Ex: \`\`\` something \`\`\`. This makes it easier to help you. I've updated both of your posts already.

The error you're seeing is due to an incorrect `token` value specified in the vuegraf.json file.

**Reply by S54Mpower** [reply]
Sorry, forgot to wrap them, yes. 

I started the whole setup from the scratch and made sure token is correct in below command:
`docker exec influxdb influx apply -f /var/lib/influxdb2/influx_dashboard.json --org vuegraf --force yes -t <my-influx-token>`

also, added the token in the vuegraf.json as well, still getting the 
`InfluxDBCkuebtError: 401: {"code":"unauthorized","message":"Unauthorized"}`

**Comment by S54Mpower** [comment]
Please see below my vuegraf.json:

`"influxDb": {`
		`"host": "10.0.4.132",`
		`"port": 8086,`
        `"user": "vuegraf",`
        `"pass": "*****",`
        `"database": "vuegraf",`
        `"reset": false,`
        `"ssl_enable": false,`
        `"ssl_verify": true,`
        `"token": "8uQ9h9jNEXhSJXtbusIAsTNoRG36JpD6ihXhyMijP9b_mUsTvRT0-rhXecEwSP9qHkERKbIKq1R2gOVJeQsA_A=="`

    },
    "updateIntervalSecs": 60,
    "detailedDataEnabled": "true",
    "accounts": [
        {
            "name": "Penthouse",
            "email": "*****",
            "password": "*****",
            "devices": [
                {
                    "name": "Right Panel",
                    "channels": [
                        "Stednjak",
                        "PC_IT",
                        "Friz_Pecnica_TV",
                        "Perilica_sudja",
                        "Masine_roba",
                        "Rasvjeta",
                        "Uticnice",
                        "Rasvjeta_Napa"
                    ]
                },
                {
                    "name": "Left Panel",
                    "channels": [
                        "none",
                        "none",
                        "none",
                        "none",
                        "Mikrovalna",
                        "Klime",
                        "Garaza",
                        "Rasvjeta"
                    ]
                }
            ]
        }
    ]
}

**Comment by S54Mpower** [comment]
Fixed, 
confusion is between the provided vuegraf.json in the attached code and in the example in the readme. Was missing the org, url and bucket. 

 Now working

**Reply by jertel** [reply]
I fixed the sample, thanks for pointing that out!

**Reply by S54Mpower** [reply]
Thank you for all the effort!
